docs/rst/installation/linux_source_installation.rst

```diff
@@ -0,0 +1,118 @@
+.. _linux_source_installation:
+
+Linux installation from sources
+===============================
+
+This section explains how to build Vulcanexus in Ubuntu Focal.
+Please, start following the `building instructions for ROS 2 Galactic Geochelone <https://docs.ros.org/en/galactic/Installation/Ubuntu-Development-Setup.html>`_ following the steps in the `System setup <https://docs.ros.org/en/galactic/Installation/Ubuntu-Development-Setup.html#system-setup>`_ and the `Get ROS 2 code <https://docs.ros.org/en/galactic/Installation/Ubuntu-Development-Setup.html#get-ros-2-code>`_ sections.
+
+Get Vulcanexus code
+-------------------
+
+Following ROS 2 building instructions, the ROS 2 code has been already downloaded into ``ros2_galactic/src`` folder.
+Vulcanexus provides several tools and packages to the ROS 2 distribution that have to be included in the workspace:
+
+.. code-block::
+
+    cd ~
+    mv ros2_galactic vulcanexus_galactic
+    cd vulcanexus_galactic
+    wget https://raw.githubusercontent.com/eProsima/vulcanexus/galactic/vulcanexus.repos
+    wget https://raw.githubusercontent.com/eProsima/vulcanexus/galactic/colcon.meta
+    vcs import --force src < vulcanexus.repos
+
+Install Vulcanexus dependencies
+-------------------------------
+
+First, follow the `Install dependencies using rosdep <https://docs.ros.org/en/galactic/Installation/Ubuntu-Development-Setup.html#install-dependencies-using-rosdep>`_ section of ROS 2 installation guide.
+Then, some additional dependencies which are required for the Vulcanexus distribution must be installed.
+Start by adding the Qt 5.15 repository:
```

Comment:
I think we should comment why is this necessary. Actually, the same applies to the binaries installation